gradle.properties

```diff
@@ -1,13 +1,13 @@
 GROUP=com.apollographql.android
```

Comment:
We're good to change that to `com.apollographql` according to https://issues.sonatype.org/browse/OSSRH-27585